server/src/schema.graphql

```diff
@@ -38,12 +39,19 @@ type Settings {
   contracts: SettingsContracts!
 }
 
+type IpfsContent {
```

Comment:
@alex-kooper I'm using GraphQL purely as a way to share types between the server and client here. There's probably a better way to do this, but we already have this setup to generate types that both client and server applications can understand. You'll see in the server's handler code as well as the client's file upload form code that they both reference this type in order to annotate what's expected to come back from the server.

Reply:
I think that is fine, for now